amundsen_application/static/css/_labels.scss

```diff
@@ -0,0 +1,19 @@
+.label-danger {
```

Comment:
Did you not have to `import 'variables'`? Interesting... we might might be able to do remove some unnecessary imports if that is the case.

Reply:
I did not have to import it.  I can't trace why at the moment but until we know exactly why this worked I may just add in the `import 'variables'`. The reason for that is that I don't want to leverage the fact that this works on an unintended behavior that we haven't tracked down yet, and then we can clean up when we have the bandwidth to look at it. 

Would you like me to file an internal ticket, or might you take a look when you add in the system for our new colors?